ang/civicase/case/services/cases-utils.service.js

```diff
@@ -17,11 +17,29 @@
       ContactsCache.add(contacts);
     };
 
+    /**
+     * Process all case clients contacts ids.
```

Comment:
It returns all case clients, so instead of saying `process`. Can we say "Get all case clients contact ids." or something similar.